yt_dlp/extractor/zdf.py

```diff
@@ -316,12 +355,32 @@ def _extract_entry(self, url, player, content, video_id):
             'timestamp': unified_timestamp(content.get('editorialDate')),
             'thumbnails': thumbnails,
             'chapters': chapters or None,
+            'episode': title,
+            **traverse_obj(content, ('programmeItem', 0, 'http://zdf.de/rels/target', {
+                'series_id': ('http://zdf.de/rels/cmdm/series', 'seriesUuid', {str}),
+                'series': ('http://zdf.de/rels/cmdm/series', 'seriesTitle', {str}),
+                'season': ('http://zdf.de/rels/cmdm/season', 'seasonTitle', {str}),
+                'season_number': ('http://zdf.de/rels/cmdm/season', 'seasonNumber', {int_or_none}),
+                'season_id': ('http://zdf.de/rels/cmdm/season', 'seasonUuid', {str}),
+                'episode_number': ('episodeNumber', {int_or_none}),
+                'episode_id': ('contentId', {str}),
+            })),
```

Comment:
Currently, the usage of `merge_dicts` will overwrite these metadata fields if they already exist in the `info` dict. Do we want that? or do we want to priotize the new data? Or doesn't it matter?

Reply:
From my testing all those metadata fields were not present in the info dict, so probably does not matter

 (the merge_dicts was already there though so this was not a conscious decision on my part)